x-pack/plugins/ingest_pipelines/public/application/pipeline_editor/components/processor_settings_form/processors/gsub.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { i18n } from '@kbn/i18n';
+
+import {
+  FieldConfig,
+  FIELD_TYPES,
+  fieldValidators,
+  FormRow,
+  UseField,
+  Field,
+} from '../../../../../shared_imports';
+
+const { emptyField } = fieldValidators;
+
```

Comment:
Do you prefer to declare all the configs above the component instead of inside a `FormSchema`? I would find it easier to scan to have all the fields declaration in a single place, but that's my preference 😊